### DISCUSSION TITLE
[mini.base16] "Strong" TS style missing

### DISCUSSION THREAD

**Discussion by alezunino** [nvim-mini/mini.nvim#455] [Q&A] [answered]
Hi,
I have noticed that when using mini.base16 colors, some words lack bold style compared to other colorschemes such as catppuccin, using TreeSitter supported languages such as Markdown. This can be fixed by adding this line to base16.lua:
`  
hi('Strong',     {fg=nil,      bg=nil, attr='bold',      sp=nil})
`

I am not sure whether this is a bug or it is the expected formatting, but I prefer the way catppuccin displays in bold some words.

thanks

**Comment by echasnovski** [comment]
Hello!

I am not sure where `Strong` highlight group comes from. I've never noticed it defined.

There is [`Bold`](https://github.com/echasnovski/mini.nvim/blob/06bf1784d88b112de14ce29aa604e2ae3fe1e359/lua/mini/base16.lua#L568) highlight group which can be served same purpose.

**Comment by alezunino** [comment]
Hi!
I was not clear. This is a short markdown file where **strong** is shown in bold using the catppuccin colorscheme:
[catppuccin](https://i.postimg.cc/rFsThfgz/catppuccin.png)

This is the same file using minicyan (bold is not shown):
[minicyan](https://i.postimg.cc/L8T6wJL8/minicyan.png)

Finally, using minicyan patched (bold is ok):
https://i.postimg.cc/MGHxLdz9/minicyan-text-strong.png

This is the code I added to mini.base16:
`  hi('@text.strong',     {fg=nil,      bg=nil, attr='bold',      sp=nil})
`

In catppuccin there are several @text hl groups such as [this](https://github.com/catppuccin/nvim/blob/2e3e5ebcdc24ef0d5b14a0a999dbbe7936512c46/lua/catppuccin/groups/integrations/treesitter.lua#L96) that are not present in mini.base16.

**Reply by echasnovski** [reply, marked as answer]
I see now, thanks! 'mini.base16' only mildly interferes with tree-sitter, 'mini.hues' has more effect there. But both of them don't create `@text.strong`. I'll add it (hopefully, tomorrow).

**Reply by echasnovski** [reply]
@alezunino, this (along with `@text.emphasis` and `@text.strike`) should  now be on `main` branch. Thanks for noticing this! Markdown support is now much better!

**Reply by alezunino** [reply]
Awesome!!!! thanks for this amazing set of plugins! 

BTW, I am using mini.base16 to generate a colorscheme based on the currently configured [WezTerm](https://wezfurlong.org/wezterm/index.html) theme. Once I test it properly I will share the code.

thanks again!
